<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewEntryClass</class>
 <widget class="QDialog" name="AddNewEntryClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>321</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add New Entry</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>441</width>
     <height>304</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Title: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Title">
      <property name="placeholderText">
       <string>Type title</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Username: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Username">
      <property name="placeholderText">
       <string>Type username</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Password: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_Password">
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Type password</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="buttonGeneratePassword">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Generate secure password</string>
      </property>
      <property name="text">
       <string>Generate Password</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>URL:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_Url">
      <property name="placeholderText">
       <string>Type URL</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Notes: </string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPlainTextEdit" name="plainTextEditNotes">
      <property name="placeholderText">
       <string>Type any optional notes here</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="checkBoxShowPassword">
      <property name="text">
       <string>Show Password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_Title</tabstop>
  <tabstop>lineEdit_Username</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>buttonGeneratePassword</tabstop>
  <tabstop>plainTextEditNotes</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonGeneratePassword</sender>
   <signal>clicked()</signal>
   <receiver>AddNewEntryClass</receiver>
   <slot>GenerateNewPassword()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Title</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddNewEntryClass</receiver>
   <slot>TitleFieldChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Username</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddNewEntryClass</receiver>
   <slot>UserFieldChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Password</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddNewEntryClass</receiver>
   <slot>PasswordFieldChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxShowPassword</sender>
   <signal>toggled(bool)</signal>
   <receiver>AddNewEntryClass</receiver>
   <slot>ShowPassword()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>GenerateNewPassword()</slot>
  <slot>TitleFieldChanged()</slot>
  <slot>UserFieldChanged()</slot>
  <slot>PasswordFieldChanged()</slot>
  <slot>ShowPassword()</slot>
 </slots>
</ui>
